src/queries/CreateAccessibilityRequestQuery.ts

```diff
@@ -1,6 +1,6 @@
 import { gql } from '@apollo/client';
 
-export const CreateAccessibilityRequestQuery = gql`
+export default gql`
```

Comment:
I'm curious about this change. I thought it'd be easier to wade through all our queries if they were grouped. We have so many files in that folder. I have my IDE and my idiosyncrasies, so it might be different for you! Is it easier for you to have them each in their own file?

Reply:
I don't feel strongly about this. I changed this because we had only one case where they were grouped, and due to how our tooling is configured that meant that how the queries were imported looked different based on if they were grouped or not:

```jsx
import { DeleteAccessibilityRequestQuery } from 'queries/AccessibilityRequestQueries';
```
vs.

```jsx
import GetAccessibilityRequestQuery from 'queries/GetAccessibilityRequestQuery';
```

I think I find one query per file easier to work with (it's obvious where a query lives if you know its name), but if the team wants to move to grouping them differently, I can get on board.